packages/dx-react-scheduler/docs/reference/scheduler.md

```diff
@@ -0,0 +1,57 @@
+# Scheduler Reference
+
+The Scheduler is a root container component designed to process and display the specified data. The Scheduler's functionality (data visualization and processing) is implemented in several plugins specified as child components.
+
+## Import
+
+Use the following statement to import a plugin with embedded theme components:
+
+```js
+import { Scheduler } from '@devexpress/dx-react-scheduler-material-ui';
+```
+
+If you want to use custom components, you can import the themeless plugin:
+
+```js
+import { Scheduler } from '@devexpress/dx-react-scheduler';
+```
+
+## User reference
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+data | Array&lt;[AppointmentModel](#appointmentmodel)&gt; | | An array of appointment data objects.
+rootComponent | ComponentType&lt;[Scheduler.RootProps](#schedulerrootprops)&gt; | | A component that renders the root layout.
+
+## Interfaces
+
+### AppointmentModel
+
+Describes an appointment data object that should be returned by `mapAppointmentData` function.
```

Comment:
Describes an appointment data object that should be returned by `mapAppointmentData` function -> Describes an appointment data object that the `mapAppointmentData` function should return.